L1Trigger/L1TMuonEndCap/interface/Event.h

```diff
@@ -12,42 +12,43 @@
 //////////////////////////////////////////////////////////////////////////
 // ----------------------------------------------------------------------
 //////////////////////////////////////////////////////////////////////////
-
-struct Event
-{
-  
-  Double_t trueValue;
-  Double_t predictedValue;
-  Double_t DTPt;
-  Double_t CSCPt;
-  Double_t tmvaPt;
-  Double_t tmvaPt1;
-  Int_t Mode;
-  Int_t Quality;
-  
-  static Int_t sortingIndex;
-  Int_t id;    
-  std::vector<Double_t> data;         
-  
-  bool operator< (const Event &rhs) const
-  {
-    return data[sortingIndex] < rhs.data[sortingIndex];
-  }
-  
-  void outputEvent()
+namespace emtf{
+  struct Event
   {
-    std::cout << "trueValue = " << trueValue << std::endl;
-    std::cout << "predictedValue = " << predictedValue << std::endl;
-    std::cout << "id = " << id << std::endl;
-    for(unsigned int i=0; i<data.size(); i++)
-      {
-	std::cout << "x"<< i << "=" << data[i] << ", ";
-      }
-    std::cout << std::endl;
     
-  }
-  
-  void resetPredictedValue(){ predictedValue = 0; }
-};
+    Double_t trueValue;
+    Double_t predictedValue;
+    Double_t DTPt;
+    Double_t CSCPt;
+    Double_t tmvaPt;
+    Double_t tmvaPt1;
+    Int_t Mode;
+    Int_t Quality;
+    
+    static Int_t sortingIndex;
```

Comment:
Method sortEventVectors uses the compareEvents which depends on static Event::sortingIndex. I agree the solution is not the best, but the code has been validated as is. 
We are in the process of integrating a re-written L1TMuonEndCap emulator, so this code will 
be replaced.  
@davidlange6 Are you fine leaving this as is, and we make sure static is not used in the new code.